stl/inc/bit

```diff
@@ -94,18 +96,117 @@ _NODISCARD constexpr _Ty rotr(const _Ty _Val, const int _Rotation) noexcept {
     }
 }
 
-template <class _Ty, enable_if_t<_Is_standard_unsigned_integer<_Ty>, int> _Enabled>
-_NODISCARD constexpr int countl_zero(const _Ty _Val) noexcept {
+// Implementation of popcount without using specialized CPU instructions.
+// Used at compile time and when said instructions are not supported.
+template <class _Ty>
+_NODISCARD constexpr int _Popcount_helper(_Ty _Val) noexcept {

```

Comment:
Still `_helper` ? I thought `_fallback` was proposed, but I can't find that comment.

Or if `_fallback` is a bad name too, maybe `_generic`, or `_safe`, or `_any_cpu`.

`_helper` has no meaning.